pydicom/tests/test_encoders.py

```diff
@@ -317,7 +286,7 @@ def test_bytes_specific(self):
         out = self.enc.encode(
             self.bytes, encoding_plugin='pydicom', **self.kwargs
         )
-        assert 21098 == len(out)
+        assert len(out) == 21350
```

Comment:
The encoded lengths changed because the temporary solution was encoding the entire pixel array as 1D array (i.e. one giant row), whereas now it gets encoded properly on a row-by-row basis which adds some overhead.